go.mod

```diff
@@ -6,6 +6,8 @@ replace cloud.google.com/go => cloud.google.com/go v0.102.1
 
 replace github.com/vmware-tanzu/tanzu-cli/test/e2e/framework => ./test/e2e/framework
 
+replace github.com/vmware-tanzu/tanzu-plugin-runtime => github.com/vuil/tanzu-plugin-runtime a454868
```

Comment:
to be updated once runtime side changes are merged

Reply:
I wasn't aware you could fix a sha when doing a replace.  Nice!